DEFAULT_CONFIG.json5

```diff
@@ -297,6 +297,17 @@
 //        demo2: {
 //          key_expr: "demo/memory2/**",
 //          volume: "memory",
+//          /// Storage manager plugin handles metadata in order to ensure convergence of distributed storages configured in Zenoh.
+//          /// Metadata includes the set of wild card updates and deletions (tombstones).
+//          /// Once the samples are guaranteed to be delivered, the metadata can be garbage collected.
+//          garbage_collection_config: {
+//            /// The garbage collection event will be periodic with this duration.
+//            /// The duration should be specified in seconds.
+//            garbage_collection_period: 30,
```

Comment:
We are already in the *garbage_collection* section, let's simply call it *period*.